### ISSUE TITLE
add refresh button for querying for new runs without completely reloading app

### ISSUE THREAD

**Issue by gfrogat** [Caleydo/malevo#227]

**Comment by thinkh** [comment]
You mean, when adding new run JSON files to the backend, they should be available in the select2?

**Comment by thinkh** [comment]
In the simplist case the refresh button can load the run list and replace it in the frontend. A more sophisticated option would be a websocket connect and receiving new runs via "push" event.
In my opinion we should keep it simple. ;-)

**Comment by gfrogat** [comment]
yes, querying the run endpoint and updating the select2 would be enough.

websockets would be nice for live-updates but we already iceboxed that idea some time ago.
